### PR TITLE
(MAINT) Update Changelog for 1.1.4

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

**Comment by melissa** [comment]
@mckern @underscorgan @cprice404 @camlow325 should we hold off on the 1.1.4 release, or can I get this out the door?

**Comment by cprice404** [comment]
I don't think there's anything on our end that would block cutting a release, whenever is best for y'all.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 1.1.4 - 2016-11-10
+  * (EZ-102) Install SysV files in addition to SystemD files
```
Comment (underscorgan) [inline CHANGELOG.md]: This was jsut for deb* right? Or did it also change for rpm

**Comment by melissa** [comment]
@underscorgan good point. CHANGELOG message and date have been updated

**Pull request merged**